### ISSUE TITLE
snaplen being set to a value that Debian 10's libpcap doesn't like

### ISSUE THREAD

**Issue by timwoj** [JustinAzoff/pcap_simplify#6]
This has happened a couple of times now with pcaps generated from oss-fuzz testcases:

```
[#3] scripts.base.protocols.smb.smb1-OSS-fuzz-54883 ... failed
  % 'zeek -b -C -r $TRACES/smb/smb1-OSS-fuzz-54883.pcap /zeek/testing/btest/.tmp/scripts.base.protocols.smb.smb1-OSS-fuzz-54883/smb1-OSS-fuzz-54883.test' failed unexpectedly (exit code 1)
  % cat .stderr
  fatal error: problem with trace file /zeek/testing/btest/Traces/smb/smb1-OSS-fuzz-54883.pcap (invalid file capture length 524288, bigger than maximum of 262144)
```
(from https://cirrus-ci.com/task/5437331591659520)

It's only happening on Debian 10. I ran `editcap` and fixed it on another pcap but we should probably fix `pkt-to-pcap` to not make pcap files like this.

**Comment by timwoj** [comment]
Scratch that. I thought I had fixed a pcap previously but it was just that between my two tests we disabled Debian 10 builds for PRs and so my change didn't actually get tested.

**Comment by JustinAzoff** [comment]
Which tool generated that pcap? That value is 512*1024 I don't know where it's coming from exactly..

**Comment by timwoj** [comment]
`tcpdump` did it. I can set it manually via tcpdump, I suppose. Not sure why it's suddenly setting it that high though when we've generated other pcaps without it doing that.

**Comment by JustinAzoff** [comment]
ah I'm a bit confused then.  So we had a simplified .pkt file output from the fuzzer right? and then used pkt-to-pcap to turn it back into a .pcap?

**Comment by awelzel** [comment]
I've opened #7 as another take on this. I've not been a fan of using `tcpdump` and now was the time :-)

**Comment by JustinAzoff** [comment]
fixed by #7, hopefully :-)
